go/src/koding/remoteapi/client/collaboration/post_remote_api_collaboration_add_responses.go

```diff
@@ -49,7 +49,7 @@ func NewPostRemoteAPICollaborationAddOK() *PostRemoteAPICollaborationAddOK {
 
 /*PostRemoteAPICollaborationAddOK handles this case with default header values.
 
-Request processed succesfully
```

Comment:
https://github.com/koding/koding/blob/master/scripts/api-generator.coffee#L169

if you change here, all will be fixed